.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = */migrations/*
```

Comment:
Let's exclude setup.py and manage.py as well. We do not need to cover these files by unit tests